ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -670,13 +670,15 @@ if tensorrt_support_is_available
 
   shared_library('nnstreamer_filter_tensorrt',
     filter_sub_tensorrt_sources,
+    link_args: '-L/usr/local/cuda-12.1/targets/x86_64-linux/lib/stubs',
```

Comment:
Nope. You cannot and should not do this. You cannot guarantee that everyone will install tensorrt binraies to this hardcoded directory (think of other distro of Linux users or even.. Arm/RISC-V users).

Library linking path should be determined dynamically at build-time, usually by pkg-config or library-finding mechanism of meson. Refer to other library finding codes in meson.build files.